src/controllers/FlatController.ts

```diff
@@ -5,7 +5,7 @@ export class FlatController {
 
   @Path("hello")
   @GET
-  public async helloWorld(): Promise<String> {
-    return 'Hello World!';
+  public async helloWorld(): Promise<string> {
+    return "Hello World!";
```

Comment:
do we want dobule quotes then?

Reply:
This is a new discussion with ESLint ;), I would suggest to create an issue for that ;)

Reply:
Allright, I'll approve it but I'll add soon the single quote rule 😈